tests/bsim/bluetooth/host/compile.sh

```diff
@@ -60,5 +60,8 @@ app=tests/bsim/bluetooth/host/privacy/device compile
 
 app=tests/bsim/bluetooth/host/security/bond_overwrite_allowed compile
 app=tests/bsim/bluetooth/host/security/bond_overwrite_denied compile
+# TODO: Add back test when #XXXXX is fixed
```

Comment:
Misisng Github issue reference?

Reply:
Yes sorry, actually I already created the PR for that bug (https://github.com/zephyrproject-rtos/zephyr/pull/58470). I think I'll just remove that `TODO` after the new PR is merged.